patches/chrome-browser-resources-md_extensions-manager.html.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/md_extensions/manager.html b/chrome/browser/resources/md_extensions/manager.html
+index 797b13e7c074615d6234178ca4902fc3f49c0252..5c7f755cec3a9bd15414f1edf793f64ef44c82c1 100644
+--- a/chrome/browser/resources/md_extensions/manager.html
++++ b/chrome/browser/resources/md_extensions/manager.html
+@@ -2,7 +2,7 @@
+ 
+ <link rel="import" href="chrome://resources/cr_elements/cr_drawer/cr_drawer.html">
+ <link rel="import" href="chrome://resources/cr_elements/cr_lazy_render/cr_lazy_render.html">
+-<link rel="import" href="chrome://resources/cr_elements/cr_toolbar/cr_toolbar.html">
++<!-- <link rel="import" href="chrome://resources/cr_elements/cr_toolbar/cr_toolbar.html"> -->
```

Comment:
instead of commenting out, should we just remove the line?

Reply:
not sure what is best for chromium rebasing - happy with whatever, so I'll remove